ResearchKit/ActiveTasks/ORKActiveStepViewController.m

```diff
@@ -362,6 +363,9 @@ - (void)finish {
     if (self.activeStep.shouldVibrateOnFinish) {
         AudioServicesPlayAlertSound(kSystemSoundID_Vibrate);
     }
+    if (self.activeStep.hasVoice && self.activeStep.finishedSpokenInstruction) {
+        [[ORKVoiceEngine sharedVoiceEngine] speakText:self.activeStep.finishedSpokenInstruction];
```

Comment:
No. It would be spoken before the next step's spoken instruction. This is used in cases where the *next* step may not be a completion step but something unrelated. Really, you want to let the user know that *this* step is finished rather than that the *next* step has begun.